scripts/bootstrap/buildbootstrapcli.sh

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env bash
```

Comment:
How are you validating your bootstrap script?  is there a docker image for an unsupported OS you're using, or something else?

Reply:
I have validated the script in various VMs and docker images (Alpine, RHEL 6 and FreeBSD)